gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpec.java

```diff
@@ -125,6 +137,21 @@ public static FlowSpec.Builder builder(URI catalogURI, Properties flowProps) {
     }
   }
 
+  /**
+   * Add property to Config (also propagated to the Properties field).
```

Comment:
You can add a line, when the property is being added, config and configAsProperties can have different values , but they should be consistent by the time method returns.